### PR TITLE
Deprecate Notify resource

### PR DESCRIPTION
**Pull request by Lir10**

### CODE REVIEW

**Comment by Lir10** [comment]
@etabak @zgalor ptal

**Review by tzvatot — APPROVED** [review]
LGTM

**Comment by Lir10** [comment]
@etabak ptal and feel free to merge it :)

File: CHANGES.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
 
+## 0.0.351 Jan 30 2024
+- Deprecate notify resource
+
 ## 0.0.350 Jan 29 2024
 - [OCM-5802] Add `ExternalAuthConfig` to `Cluster` model.
 
-## 0.0.349 Jan 28 2024
--  [OCM-5561] Remove redundant fields from /notify_details
+## 0.0.349 Jan 28 2024Notify from /notify_details
```
Comment (tzvatot) [inline CHANGES.md:11]: @Lir10 why is this change required?

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (Lir10) [reply CHANGES.md]: good catch .. no idea where that came from.
updated

**Review by tzvatot — APPROVED** [review]
